plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -4714,6 +4716,36 @@ protected void verifyIcebergTableProperties(MaterializedResult actual)
         assertEqualsIgnoreOrder(actual.getMaterializedRows(), expected.getMaterializedRows());
     }
 
+    @Test
+    public void testGetIcebergTableWithLegacyOrcBloomFilterProperties()
+            throws Exception
+    {
+        String tableName = "test_get_table_with_legacy_orc_bloom_filter_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT 1 x, 'INDIA' y", 1);
+
+        String tableLocation = getTableLocation(tableName);
+        String metadataLocation = getLatestMetadataLocation(fileSystem, tableLocation);
+
+        TableMetadata tableMetadata = TableMetadataParser.read(new ForwardingFileIo(fileSystem), metadataLocation);
+        ImmutableMap<String, String> newProperties = ImmutableMap.<String, String>builder()
+                .putAll(tableMetadata.properties())
+                .put("orc.bloom.filter.columns", "x,y") // legacy incorrect property
+                .put("orc.bloom.filter.fpp", "0.2") // legacy incorrect property
+                .buildOrThrow();
+        TableMetadata newTableMetadata = newTableMetadata(
+                tableMetadata.schema(),
+                tableMetadata.spec(),
+                tableMetadata.sortOrder(),
+                tableMetadata.location(),
+                newProperties);
+        TableMetadataParser.overwrite(newTableMetadata, new ForwardingOutputFile(fileSystem, Location.of(metadataLocation)));
+
+        MaterializedResult actualCreateTable = computeActual("SHOW CREATE TABLE " + tableName);
+        assertThat(actualCreateTable).isNotNull();
+        assertThat(actualCreateTable.getMaterializedRows().getFirst().toString())
+                .contains(ImmutableList.of("orc_bloom_filter_columns", "orc_bloom_filter_fpp"));
```

Comment:
We could simplify like below:
```java
        assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName))
                .contains("orc_bloom_filter_columns", "orc_bloom_filter_fpp");
```

Same for TestIcebergOrcWithBloomFilters.

Reply:
https://github.com/trinodb/trino/pull/20908